src/main/webapp/data/developers.json

```diff
@@ -312,6 +317,10 @@
       "name": "Connor McDonald",
       "username": "clmcdonald"
     },
+    {
+      "name": "",
```

Comment:
I mean, just omit the `name` field entirely. You can look at a few examples in this JSON file.